signal_scanner_bot/transport.py

```diff
@@ -29,6 +29,10 @@ async def twitter_to_queue():
         follow=",".join(env.TRUSTED_TWEETERS)
     )
     async with stream_obj as stream:
+        # If we aren't supposed to be listening return nothing.
```

Comment:
I think this might need to be one layer lower, in the `async for data in stream` clause (since I think that's the loop this layer will be on for the duration of its runtime